scapy/layers/l2.py

```diff
@@ -424,6 +424,8 @@ def l2_register_l3_arp(l2, l3):
 
 conf.neighbor.register_l3(Ether, ARP, l2_register_l3_arp)
 
+class ERSPANfake(Packet):
```

Comment:
I think that these modifications should be integrated in to `scapy/contrib/ersspan.py`. Could you also add examples and unit tests?